packages/appstore/webpack.config.js

```diff
@@ -139,7 +139,24 @@ module.exports = function (env) {
                 },
                 {
                     test: /\.svg$/,
-                    exclude: /node_modules|public\//,
+                    issuer: /\/packages\/wallets\/.*(\/)?.*.scss/,
```

Comment:
The `issuer` ensures that the issuer (the parent file who is importing the loaded file) is a CSS/SCSS file and emits the SVG into `appstore/wallets/public`. This should not be passed into `svg_loaders` as it will transform it into a React component and inline it within the CSS `url`

Reply:
<img width="1840" alt="Screenshot 2023-08-29 at 8 32 36 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/9e77b69e-bf34-4325-815a-c22b406e2e22">